src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java

```diff
@@ -135,6 +137,7 @@ private static void addAnnotation(Project project, LiveFinding finding, TextRang
 
     if (finding instanceof LiveIssue) {
       intentionActions.add(new MarkAsResolvedAction((LiveIssue) finding));
+      intentionActions.add(new ReopenIssueAction((LiveIssue) finding));
```

Comment:
Is this possible? Do we annotate issues that are marked resolved? I'm not sure we want to bring attention on them for users

Reply:
I'll confirm this with PM, in the meantime I'm removing all the annotation part, just having them visible in the list is enough probably

Reply:
It should not be annotated